src/test/acceptance/features/cookies.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Cookies details page
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to view the cookies used by the site
+
+  Scenario: The cookie back link is not shown if we navigate directly to it
```

Comment:
I think this would be better as `The back link on the Cookies page is not shown if we navigate to it directly`.

Reply:
Done.

Reply:
Done.